packages/core/src/analytics/integrations/Omnitracking/Omnitracking.js

```diff
@@ -215,26 +218,56 @@ class Omnitracking extends Integration {
         );
       }
       case analyticsTrackTypes.TRACK: {
-        precalculatedParameters = this.getPrecalculatedParametersForEvent(data);
+        const eventMapperFn = trackEventsMapper[data.event];
 
-        if (!this.validateTrackingRequisites()) {
-          logger.warn(
-            `[Omnitracking] - Event ${data.event} tried to track without an unique view id.
-                Consider tracking an event only after a page event.
-                This behaviour is not recommended and will not be supported in upcoming versions`,
+        if (eventMapperFn && !isFunction(eventMapperFn)) {
+          throw new Error(
```

Comment:
This error seems to imply that the user is making a configuration mistake, however the `trackEventsMapper` is internal. Maybe the best here would be to have a unit test that checked if all values from all keys of `trackEventsMapper` are a function?

Reply:
Fixed on dev-main-22-02-22. thanks 🙂